R/write.magpie.R

```diff
@@ -196,9 +197,11 @@ write.magpie <- function(x, file_name, file_folder = "", file_type = NULL, appen
         x <- x[[1]]
       }
       if (is.null(varname)) varname <- "Variable"
+      zname <- if(!is.null(zname)) zname else "Time"
       raster::writeRaster(x, filename = filePath, format = format[file_type], overwrite = TRUE,
-                          zname = "Time", zunit = zunit, varname = varname, ...)
+                          zname = zname, zunit = zunit, varname = varname, ...)
     } else if (file_type == "nc") {
+      zname <- if(!is.null(zname)) zname else "Time"
```

Comment:
I think this line is not needed if you set `zname = "Time"` by default

Reply:
Okie dokie, makes sense!